db/agents.yml

```diff
@@ -246,13 +247,6 @@ agents:
     examples:
       - Clementine 1.3.1 Clementine 1.3.1 Clementine 1.3.1
       - Clementine 1.3.1
-  - regex: '^Dalvik.+Android'
```

Comment:
NPR One android now starts with `Dalvik` ... to prevent this one from matching, I had to move it down towards the bottom of this list.